### PR TITLE
[CDAP-13879] Changes icon color and shape for Pending status in pipelines

### PR DESCRIPTION
**Pull request by tonybach**
JIRA: https://issues.cask.co/browse/CDAP-13879
Builds: https://builds.cask.co/browse/CDAP-UDUT95

### CODE REVIEW

File: cdap-ui/app/hydrator/templates/list.html [hunk 1]
```diff
@@ -71,8 +71,11 @@ <h1 class="pull-left">Pipelines</h1>
               <span>{{ ::ListController.GLOBALS.artifactConvert[pipeline.artifact.name] }}</span>
             </td>
             <td>
-              <span class="status-bubble"
-                ng-class="ListController.MyPipelineStatusMapper.getStatusIndicatorClass(pipeline.displayStatus)"></span>
+                <span class="status-bubble"
+                  ng-class="ListController.MyPipelineStatusMapper.getStatusIndicatorClass(pipeline.displayStatus)">
+                  <icon-svg name="ListController.MyPipelineStatusMapper.getStatusIndicatorIcon(pipeline.displayStatus)"></icon-svg>
```
Comment (ajainarayanan) [inline cdap-ui/app/hydrator/templates/list.html:76]: Will this handle drafts?

**Review by ajainarayanan — COMMENTED** [review]
@tonybach Are we changing the pipeline summary view to show the right icon for pending runs? Right now when I start the run the run info shows pending icon but the summary shows the running(or succeeded) icon in the graph. 
I understand its an existing issue but just checking if we are addressing it in this PR

**Comment by tonybach** [comment]
@ajainarayanan Hmm I don't think it was part of the scope for this PR, since the issue you mentioned has a separate JIRA.

Have added a commit to fix icon for drafts.

File: cdap-ui/app/cdap/services/StatusMapper.js [hunk 2]
```diff
@@ -47,21 +47,27 @@ function lookupDisplayStatus (systemStatus) {
...
     return 'status-light-red';
-  } else if (displayStatus === 'Draft') {
-    return 'status-outline-grey status-thin';
   } else {
     return 'status-light-grey';
   }
 }
 
+function getStatusIndicatorIcon (displayStatus) {
+  if (displayStatus === 'Pending' || displayStatus === 'Draft') {
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/services/StatusMapper.js]: Sorry I missed this earlier but can we use the `StatusMapper` for the status of the pipeline? Will reduce the amount of things we hardcode.

**Review by ajainarayanan — DISMISSED** [review]
One minor comment. Otherwise 👍

**Review by ajainarayanan — APPROVED** [review]

**Comment by tonybach** [comment]
Green build: https://builds.cask.co/browse/CDAP-UDUT95-3

**Pull request merged**